src/settings/RequestCancellationReasons.js

```diff
@@ -30,7 +30,12 @@ class RequestCancellationReasons extends React.Component {
         visibleFields={['name', 'description', 'publicDescription']}
         hiddenFields={['lastUpdated', 'numberOfObjects']}
         columnMapping={{
-          name: <FormattedMessage id="ui-circulation.settings.cancelReasons.labelShort" />,
+          name: (
+            <React.Fragment>
+              <FormattedMessage id="ui-circulation.settings.cancelReasons.labelShort" />
+              <Asterisk />
+            </React.Fragment>
```

Comment:
I don't think this is correct.  What is needed is a `required` attribute on the Field.  See https://github.com/folio-org/ui-inventory/blob/master/src/edit/InstanceForm.js#L484 for an example.

Reply:
It's not possible, because in both cases FormattedMessage component which renders label does not belong to any Field component as label property.

Reply:
I see what you mean.
![image](https://user-images.githubusercontent.com/152960/63119861-1bc9dd00-bf6f-11e9-9a83-8bea76a90998.png)

I'm not entirely comfortable with how this implementation is different from how the `required` attribute is used on `<Field>` tags.  I'm going to take this into the "#stripes" channel on Slack for advice.